hrms/payroll/doctype/payroll_entry/payroll_entry.py

```diff
@@ -923,7 +923,11 @@ def make_bank_entry(self, for_withheld_salaries=False):
 
 					salary_slip_total -= salary_detail.amount
 
-			salary_slip_total -= flt(salary_detail.get("total_loan_repayment"))
+		total_loan_repayment = sum(
+			flt(slip.get("total_loan_repayment", 0))
+			for slip in {slip["name"]: slip for slip in salary_slips}.values()
```

Comment:
This should work?

```suggestion
			for slip in salary_slips
```

Reply:
> This should work?

Already tested because `for slip in salary_slips` iterates over each row, including duplicates for the same salary slip.
Using `{slip["name"]: slip for slip in salary_slips}.values()` filters out duplicate entries by keying each salary slip by its unique "name", so `total_loan_repayment` is only counted once per unique slip.

Reply:
Add it in a separate variable `unique_salary_slips` for readability?